sdk/config.go

```diff
@@ -0,0 +1,124 @@
+package sdk
+
+import (
+	"os"
+	"strconv"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+type HeliosConfig struct {
+	serviceName       string
+	apiToken          string
+	sampler           trace.Sampler
+	collectorEndpoint string
+	collectorPath     string
+	environment       string
+	commitHash        string
+}
+
+// Keys and their matching env vars
+const samplingRatioKey = "samplingRatio"
```

Comment:
we can use `viper` to work with env vars and allow other configurations - but probably we should stay native

Reply:
I prefer staying native